plan/plan.go

```diff
@@ -260,3 +261,27 @@ func normalizeDNSName(dnsName string) string {
 	}
 	return s
 }
+
+func CompareBoolean(defaultValue bool, name, current, previous string) bool {
+	var err error
+
+	v1, v2 := defaultValue, defaultValue
+
+	if previous != "" {
+		v1, err = strconv.ParseBool(previous)
+		if err != nil {
+			log.Errorf("Failed to parse previous property [%s]: %v", name, previous)
```

Comment:
If we decide that it is safe to ignore the error, I would rather not do anything at all and just have `v1, err = strconv.ParseBool(previous)`. As long as it is documented in the method comment, of course.

Reply:
I think it's better at least show Warning / Debug in logs, but OK it's not big thing